src/comic-page/comic-page.service.ts

```diff
@@ -52,6 +53,14 @@ export class ComicPageService {
     return comicPagesData;
   }
 
+  async getComicPagesForIssue(comicIssueId: number): Promise<ComicPage[]> {
+    return await this.prisma.comicPage.findMany({
+      where: {
+        comicIssueId,
+      },
+    });
+  }
+
```

Comment:
Here you should add `walletAddress` parameter and check if the wallet is eligible to see ALL pages or only ones that are previewable. Pick that logic from the `findOneProtected`

Reply:
Yeah, sorry, forgot to do that.